frontend/src/pages/ExploreData/OnboardingSteps.tsx

```diff
@@ -4,7 +4,7 @@ import covidClick from '../../assets/screengrabs/covidClick.mp4'
 export function getOnboardingSteps(pageIsWide: boolean) {
   const steps = [
     onboardingStep(
-      '.covid-dropdown-topic',
+      '#madlib-box',
```

Comment:
The `<Onboarding />` components gets rendered before `<MadlibUI />` finishes parsing the query params and selects the "COVID-19" dropdown. This is why the `.covid-dropdown-topic` CSS class is not available initially and gets skipped to the final step.

The `madlib-box` ID is more consistent because it is already available when the page is rendered.

https://github.com/SatcherInstitute/health-equity-tracker/blob/168ab7eed85a7d971df6d56c3b153c1448676c01/frontend/src/pages/ExploreData/MadLibUI.tsx#L104

Reply:
wow, great work debugging the issue. we really appreciate your help!